File: eBiblioteka.Servisi/Database/Clanarina.cs
using System;
using System.Collections.Generic;

namespace eBiblioteka.Servisi.Database;

public partial class Clanarina
{
    public int ClanarinaId { get; set; }

    public string StatusClanarine { get; set; } = null!;

    public DateTime DatumUplate { get; set; }

    public DateTime DatumIsteka { get; set; }

    public int? KorisnikId { get; set; }

    public int? TipClanarineId { get; set; }

    public virtual Korisnik? Korisnik { get; set; }

    public virtual TipClanarine? TipClanarine { get; set; }
}

File: eBibliotekaUI/ebiblioteka_admin/lib/models/clanarina.dart
import 'package:json_annotation/json_annotation.dart';

part 'clanarina.g.dart';

@JsonSerializable()
class Clanarina {
  int? clanarinaId;
  String? statusClanarine;
  DateTime? datumUplate;
  DateTime? datumIsteka;
  int? korisnikId;
  int? tipClanarineId;

  Clanarina({
    this.clanarinaId,
    this.korisnikId,
    this.tipClanarineId,
    this.statusClanarine,
    this.datumUplate,
    this.datumIsteka,
  });

  factory Clanarina.fromJson(Map<String, dynamic> json) =>
      _$ClanarinaFromJson(json);

  Map<String, dynamic> toJson() => _$ClanarinaToJson(this);
}
